introduction.md

```diff
@@ -0,0 +1,5 @@
+Name : G.Muralidhar Reddy	
```

Comment:
JavaScript

On Tue, Aug 18, 2015 at 11:10 PM, Code School Kiddo <
notifications@github.com> wrote:

> In introduction.md
> <https://github.com/deadlyvipers/dojo_rules/pull/6441#discussion_r37328725>
> :
>
> > @@ -0,0 +1,5 @@
> > +Name : G.Muralidhar Reddy	
>
> Looks good! Could you also please mention your favorite Code School path
> in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or
> Electives.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/deadlyvipers/dojo_rules/pull/6441/files#r37328725>.
>



-- 
*G.Muralidhar reddy*